### PR TITLE
Remove annoying recurring print from `chatroom.nut`

### PR DESCRIPTION
**Pull request by JMM889901**
First commit is upload from vanilla

Adds `chatroom.nut` from `englishclient_frontend` and modifies it.

### CODE REVIEW

**Review by ASpoonPlaysGames — APPROVED** [review]
Holy shit yes please thank you oh my god (also, code looks good)

**Comment by ASpoonPlaysGames** [comment]
> First commit is upload from vanilla

you should mention which vpk this is from

**Comment by JMM889901** [comment]
> you should mention which vpk this is from

read first commit description :)

**Review by F1F7Y — CHANGES_REQUESTED** [review]
I know I'm nitpicking but imo it would be better to just comment it out instead of removing it out right

**Review by uniboi — APPROVED** [review]
praise the lord

> you should mention which vpk this is from

click the commit bruh it's right there

**Comment by ASpoonPlaysGames** [comment]
> click the commit bruh it's right there

(this is for gecko not for me)

**Comment by JMM889901** [comment]
> I know I'm nitpicking but imo it would be better to just comment it out instead of removing it out right

👍 done

**Review by F1F7Y — APPROVED** [review]
Changes look good

**Comment by GeckoEidechse** [comment]
For completeness sake, maybe add sample log snippet that shows before/after change. That sorta stuff helps a lot with figuring out which PRs should get priority ;)

**Comment by JMM889901** [comment]
image

before^ 

after is kinda hard to show since its just the absence of those prints

**Comment by JMM889901** [comment]
boowomp

**Review by EladNLG — APPROVED** [review]
oh my god

File: Northstar.Client/mod/scripts/vscripts/ui/chatroom.nut [hunk 1]
```diff
@@ -308,7 +308,7 @@ void function UICodeCallback_ShowUserInfo( string hardware, string userId )
 {
 	Signal( uiGlobal.signalDummy, "StopUserInfoLookups" )
 
-	printt( "Showing user info for UID " + userId + " on hardware " + hardware )
+	//printt( "Showing user info for UID " + userId + " on hardware " + hardware )
```
Comment (GeckoEidechse) [inline Northstar.Client/mod/scripts/vscripts/ui/chatroom.nut:311]:
```suggestion
	// printt( "Showing user info for UID " + userId + " on hardware " + hardware )
```

**Comment by GeckoEidechse** [comment]
Merging based on previous reviews. Made sure to commit vanilla file to `main` separately first.

**Pull request merged**